.github/workflows/account_deployment.yml

```diff
@@ -1,7 +1,7 @@
 name: account-deployment
 on:
   schedule:
-      interval: "monthly"
+    - cron: 0 10 1 * *
```

Comment:
I think you need to quote the cron-expression

Reply:
You are right, tnx